Misc/NEWS.d/next/IDLE/2017-09-18.bpo-31507.dgs67n.rst

```diff
@@ -0,0 +1 @@
+Adds docstring to email.utils.parseaddr
```

Comment:
Remove this.  We don't need doc entries for news items (I'll add the skip news label), and in any case IDLE is the wrong section :)

Reply:
I figured that this didn't a news entry. But I had no idea how to add the skip news label and I wasn't sure where else to add an entry other than IDLE(a quick file search showed such entries in IDLE). You went beyond and above to help me with this, thank you 👍

Reply:
No problem.  This is part of nurturing new contributors :)